src/javascript/binary/websocket_pages/user/account/authenticate.js

```diff
@@ -1,18 +1,27 @@
-const Client       = require('../../../base/client');
-const BinarySocket = require('../../socket');
+const Client              = require('../../../base/client');
+const BinarySocket        = require('../../socket');
+const DocumentUploader    = require('binary-document-uploader');
+const showLoadingImage    = require('../../../base/utility').showLoadingImage;
+const localize            = require('../../../base/localize').localize;
+const displayNotification = require('../../../base/header').displayNotification;
+const Url                 = require('../../../base/url');
```

Comment:
can we order these based on how far they are from current file? e.g. line 1 and line 3 should swap, line 4 should go to the end (alphabetic ordering of pathname, `utility` goes after `url`, and line 6 should go to line 4